Tests/MapboxMapsTests/Style/Generated/IntegrationTests/Layers/ModelLayerIntegrationTests.swift

```diff
@@ -1,6 +1,6 @@
 // This file is generated
 import XCTest
-@_spi(Experimental) @testable import MapboxMaps
```

Comment:
this file is generated using [mapbox-maps-stylegen](https://github.com/mapbox/mapbox-maps-stylegen), we should instead remove experimental key from the spec [here](https://github.com/mapbox/mapbox-maps-stylegen/blob/12fd4b5d234043faab01d8305385ccbe9c00a386/style-spec.js#L233)